Explorer/Assets/DCL/Audio/AvatarAudioPlaybackController.cs

```diff
@@ -16,6 +19,17 @@ public class AvatarAudioPlaybackController : MonoBehaviour
         [SerializeField] private Animator AvatarAnimator;
         [SerializeField] private AvatarAudioSettings AvatarAudioSettings;
 
+        private const float WALK_INTERVAL_SEC = 0.37f;
+        private const float JOG_INTERVAL_SEC = 0.31f;
+        private const float RUN_INTERVAL_SEC = 0.25f;
+        private const float JUMP_INTERVAL_SEC = 0.25f;
+        private const float LAND_INTERVAL_SEC = 0.25f;
```

Comment:
Might be worth exposing those values in a scriptable object if we intend to edit them in the future, otherwise it's okay like this

Reply:
yes, I'm actually doing that in another branch as I extracted a lot of this functionality into another component to handle all anim events (not only sounds)